api-server/rockcraft.yaml

```diff
@@ -1,19 +1,19 @@
-# Dockerfile: https://github.com/kubeflow/pipelines/blob/2.0.5/backend/Dockerfile
+# Dockerfile: https://github.com/kubeflow/pipelines/blob/2.2.0/backend/Dockerfile
 name: api-server
 summary: An image for using Kubeflow pipelines API
 description: An image for using Kubeflow pipelines API
-version: "2.0.5"
+version: "2.2.0"
 base: ubuntu@20.04
 license: Apache-2.0
 platforms:
   amd64:
 run-user: _daemon_
 services:
   apiserver:
-    summary: "pipelines api-server service"
     override: merge
-    command: /bin/apiserver --config=/config --sampleconfig=/config/sample_config.json -logtostderr=true
+    summary: "pipelines api-server service"
     startup: disabled
+    command: /bin/apiserver --config=/config --sampleconfig=/config/sample_config.json -logtostderr=true
```

Comment:
Should we add the [loglevel](https://github.com/kubeflow/pipelines/blob/2.2.0/backend/Dockerfile#L86)?

Reply:
Added